assets/logos/README.md

```diff
@@ -0,0 +1,10 @@
+ # Installing logos
+
+ - Copy all images to the specified location:
+
+        [glassfish_directory]/domains/domain1/docroot/logos/
+
+
+- Configure your Dataverse installation to make use of the supplied `footer_logos_rds.html` file:
+
+        curl -X PUT -d 'footer_logos_rds.html' [dataverse_web_address]/api/admin/settings/:FooterCustomizationFile
```

Comment:
right now we are using file based settings so this shouldn't be done via API. I think I overlooked it on other pr.

Reply:
Okay, so how this should be done properly?
I took this method from the Dataverse docs, I don't know what exactly differs in our case.

Reply:
We are using file based settings that are in "dataverse.default.properties" normally you move those to ~/.dataverse and rename it to dataverse.properties then you edit the values. so in this case `FooterCustomizationFile=footer_logos_rds.html`.

Reply:
May I ask you for some further assistance?

I've copied the `dataverse.default.properties` to `~/.dataverse` and renamed it, and the changes don't seem to take effect. Do I need to do something else as well?

Reply:
Hmm, I'm looking at the code and this is the path `System.getProperty("user.home") + "/.dataverse/dataverse.properties"` so it extracts you'r home directory and concants the strings. It also overrides the default values.

Reply:
If this doesn't help we would need to debug together.